src/PrestaShopBundle/Form/Admin/Sell/Product/Options/OptionsType.php

```diff
@@ -27,61 +27,22 @@
 
 namespace PrestaShopBundle\Form\Admin\Sell\Product\Options;
 
-use PrestaShop\PrestaShop\Core\Form\FormChoiceProviderInterface;
-use PrestaShopBundle\Form\Admin\Type\SwitchType;
 use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
-use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
-use Symfony\Component\Translation\TranslatorInterface;
 
 /**
  * This form class is responsible to generate the product options form.
  */
 class OptionsType extends TranslatorAwareType
```

Comment:
@jolelievre Please describe this BC Break :)

Reply:
@Progi1984 please check your review bot :trollface: 
I described the BC in the description about a month ago